src/components/Button.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+function Button(type, handleChange, buttonText) {
+    return (
+        <button
+        type={type}
+        className={'button'}
+        onClick={handleChange}>
+            {buttonText}
```

Comment:
Bij App.jsx geef je wel een waarde mee aan de property buttonText, maar in jouw Counters.jsx waar je het Button component 2x gebruikt doe je dit niet. Daar zit je een '-' en een '+' tussen het component in wat zou betekenen dat het gaat om de children van het component. Dit zou correct zijn als je ook children als property zou gebruiken in jouw Button component. Zo dus:

`<button>{children}</button>`